features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml

```diff
@@ -784,11 +784,15 @@
                     <Enable>true</Enable>
                     <MaxFailedAttempts>2</MaxFailedAttempts>
                 </ReCaptcha>
+                <Forced>
+                    <Enable>false</Enable>
+                </Forced>
             </Password>
             <Answer>
                 <Regex>.*</Regex>
                 <Uniqueness>false</Uniqueness>
             </Answer>
+            <MinQuestionsToAnswer></MinQuestionsToAnswer>
```

Comment:
Better to have a default value. Ex: there is no limit = 0. 

Also, let's add a description to this property. Explaining what it is, and special cases(ex: 0)